src/integrationTest/java/uk/gov/hmcts/reform/waworkflowapi/controllers/CreateTaskTest.java

```diff
@@ -58,118 +42,59 @@ void setUp() {
         when(authTokenGenerator.generate()).thenReturn(BEARER_SERVICE_TOKEN);
     }
 
-    @DisplayName("Should create task with default due date and 201 response")
+    @DisplayName("Should evaluate a DMN and return a 200")
     @Test
```

Comment:
the dueDate tests covered the AC's for the original ticket? Where are the tests moved too?